app/controllers/measurement_sessions_controller.rb

```diff
@@ -23,14 +23,19 @@ def show
     session = Session.find_by_url_token(params[:url_token]) or raise NotFound
 
     selected_session_ids = [session.id]
-    data = { sensorId: "Temperature-airbeam2-f (F)" }
-
-    anchor = "?selectedSessionIds=#{selected_session_ids.to_json}&data=#{data.to_json}"
+    sensorId = "Temperature-airbeam2-f (F)"
 
     if (session.type == "FixedSession")
-      redirect_to fixed_map_path(:anchor => anchor)
+      data =
+        { sensorId: sensorId,
```

Comment:
Remember to use snake case in Ruby: `sensor_id`

Reply:
These are parameters, they have to look like usual params we have in the urls.

Reply:
The right side is a ruby var ;)